src/main/webapp/i18n/en/gradeComposition.json

```diff
@@ -19,14 +19,13 @@
       },
       "id": "ID",
       "name": "Name",
-      "minGradeScale": "Min Grade Scale",
-      "maxGradeScale": "Max Grade Scale",
-      "position": "Position",
```

Comment:
revert if necessary

Reply:
this is necessary. so I will not revert it